cropper/src/test/java/com/canhub/cropper/PickImageContractTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.canhub.cropper
+
+import android.app.Activity
+import android.content.Intent
+import androidx.activity.result.ActivityResultRegistry
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.core.app.ActivityOptionsCompat
+import androidx.core.net.toUri
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class PickImageContractTest {
+
+    private val testRegistry = object : ActivityResultRegistry() {
+        override fun <I, O> onLaunch(
+            requestCode: Int,
+            contract: ActivityResultContract<I, O>,
+            input: I,
+            options: ActivityOptionsCompat?
+        ) { }
+    }
+
+    @Test
+    fun testImagePicking() {
```

Comment:
I know the library lack on consistence when we talk about tests. 

But I would request we use GIVEN/WHEN/THEN pattern, sometimes is redundant, but help people with less knowledge on the code to understand, making easier for people to contribute in the future.

=)

Reply:
please elaborate

Reply:
something in this lines:
```kotlin
    @Test
    fun `GIVEN input true, WHEN pick image, THEN action is chooser`() {
        // GIVEN
        val input = true
        val expected = Intent.ACTION_CHOOSER
        var result: Intent? = null

        // WHEN
        with(launchFragmentInContainer { ContractTestFragment(testRegistry) }) {
            onFragment { fragment -> result = fragment.pickImageIntent(input) }
        }

        // THEN
        assertEquals(expected, result?.action )
    }
```